DjangoPlugin/tracdjangoplugin/settings.py

```diff
@@ -23,6 +26,6 @@
 ]
 
 
-SECRET_KEY = str(SECRETS["secret_key"])
+SECRET_KEY = str(SECRETS.get("secret_key", ""))
```

Comment:
I'd use a non-blank default, e.g.
```suggestion
SECRET_KEY = SECRETS.get("secret_key", "django-insecure-secret")
```
Also, `str()` seems unnecessary.

Reply:
My idea was to prevent accidentally running a known secret key in production which is why I used an empty default.

Isn't it the case that Django will refuse to start if it detects an empty secret key?

Reply:
Right, sorry I was thinking we have test settings here :sweat_smile:

Reply:
but `str()` is still unnecessary

Reply:
> but `str()` is still unnecessary

Most likely yes, unless you use an integer in your secrets.json but then you had it coming 😅